chapters/object_types/instances.txt

```diff
@@ -46,24 +46,51 @@ and 16th float) as being (0, 0, 0, 1).
 | bounds   |`FLOAT32_BOX3`| axis-aligned bounding box in world-space (excluding the lights)
 |===================================================================================================
 
+[NOTE]
+.Note
+====
+Since the parameters of all other instance subtypes are supersets of the
+basic parameters listed above, it is recommend that the `transform` parameter
+should always be set in addition to any subtype specific transform parameters.
+====
+
 
 [[object_types_instance_transform]]
 ==== Transform
 
 Extension `KHR_INSTANCE_TRANSFORM`
 
 The `transform` instance subtype is a basic instance that implements the
-mandatory parameters and properties listed above. When this extension is
-supported creating an instance of unknown subtype will create an instance of
-subtype `transform` instead.
+parameters and properties listed above.
+
+[[extension_instance_transform_array]]
+The `KHR_INSTANCE_TRANSFORM_ARRAY` extension adds to the `transform` subtype
+additional types for the `transform` (and related) parameters by allowing
+arrays to do multi-instantiation. This gives applications a more efficient means
+to transform a single group many times by avoiding the need to create an entire
+object per-transform.
+
+.Additional <<api_concepts_parameters, parameters>> introduced by `KHR_INSTANCE_TRANSFORM_ARRAY` <<object_types_instance, instance>>.
+[cols="<4,<3,<11",options="header,unbreakable"]
+|==========================================================================================================
+| Name      | Type                       | Description
+| transform |`ARRAY1D` of `FLOAT32_MAT4` | array of matrices, instantiating `group` once per-element matrix
+| id        |`ARRAY1D` of `UINT32`       | <<frame_channels, extension `KHR_FRAME_CHANNEL_INSTANCE_ID`>>, optional user Id, for frame <<frame_channels, channel `instanceId`>>
+|==========================================================================================================
+
+The additional `id` array applies an instance identifier value to each element
+instance of the `transform` array. The sizes of these arrays must match. If the
```

Comment:
In other places we use a less strict requirement: the `id` array only needs to be large enough (at least as large as the `transform` array), e.g. 
> All primitive.* arrays must be at least as large as (explicitly set or implicitly derived) primitive.index. All vertex.* arrays must be large enough to be indexed by the indices in (explicitly set or implicitly derived) primitive.index

Reply:
Addressed in latest revision.